companies/the-north-face.md

```diff
@@ -0,0 +1,31 @@
+---
+name: The North Face
+blm_statements:
+  - url: https://www.facebook.com/thenorthface/videos/181553773220311/
+    date_posted: 2020-05-30
+    type: Facebook
+    raw_text: >
+      We know and acknowledge that for too long, racism has made the world unequal and unsafe. We recognize the pain and suffering racism has caused and are making a commitment to be part of the solution. We are standing up, because we owe it to everyone affected, ourselves and you.
+      
```

Comment:
```suggestion

```

Reply:
I think we can leave the rule, it's wanting this whitespace gone I think?

Reply:
It should be good to go now